app/views/common/os_selection/_pxe_loader.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= fields_for item do |f| -%>
+  <% if @operatingsystem.available_loaders(item.architecture).any? -%>
+    <% selected_attr = f.object.new_record? && item.hostgroup.present? && item.hostgroup.pxe_loader.nil? ? { :selected => @operatingsystem.preferred_loader(item.architecture) } : {} -%>
+    <%= selectable_f f, :pxe_loader,
+      @operatingsystem.available_loaders.to_a, {
+        :include_blank => blank_or_inherit_f(f, :pxe_loader),
+        :selected => item.pxe_loader
+      }.merge(selected_attr),
+      :label => _('PXE Loader'),
+      :help_inline => popover(_("DHCP filename option"), _("TFTP filename to use, select one of")  + ' ' + @operatingsystem.available_loaders(item.architecture).values.join(', ') + _('or set to blank for Non-PXE method (e.g. iPXE)'), :data => { :placement => 'top' }) %>
```

Comment:
Fixed, but what you mean by the "options should match the options shown in the select field"?